scripts/vim-patch.sh

```diff
@@ -161,29 +161,19 @@ list_vim_patches() {
 
   printf "\nVim patches missing from Neovim:\n"
 
-  # Get tags since 7.4.442.
-  local vim_tags=$(cd "${VIM_SOURCE_DIR}" && git tag --contains v7.4.442)
-
-  # Get non-versioned commits since e2719096.
-  if git log -1 --grep='.' --invert-grep > /dev/null 2>&1 ; then
-    local vim_runtime_commits=$(cd "${VIM_SOURCE_DIR}" && \
-      git log --reverse --format='%H' --grep='^patch' --grep='^updated for version' \
-        --invert-grep e2719096250a19ecdd9a35d13702879f163d2a50..HEAD)
-  else # --invert-grep requires git 2.4+
-    echo "Warning: some runtime updates may not be listed (requires git 2.4+)."
-    local vim_runtime_commits=$(cd "${VIM_SOURCE_DIR}" && \
-      git log --reverse --format='%H' --grep='Updated' e2719096250a19ecdd9a35d13702879f163d2a50..HEAD)
-  fi
+  # Get commits since 7.4.602.
+  local vim_commits=$(cd "${VIM_SOURCE_DIR}" && git log --reverse --format='%H' v7.4.602..HEAD)
```

Comment:
I think this is fine, because if it's too slow it means we're too far behind. We can bump it forward as we catch up.